src/components/LifeEventPicker.tsx

```diff
@@ -1,54 +1,287 @@
-import React from 'react';
+import React, { useState } from 'react';
 import TextField from '@material-ui/core/TextField';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
 import { makeStyles } from '@material-ui/core/styles';
 
 export default function LifeEventPicker(): JSX.Element {
   const classes = useStyles();
 
+  const [state, setState] = useState({
+    birth: true,
+    residence: false,
+    marriage: false,
+    death: false,
+    anyEvent: false,
+  });
+
+  const addBirthTextField = (): any => {
```

Comment:
functions that don't have a return value should be declared as `void`:

`function addBirthTextField(): void {...}`
or
`const addBirthTextField = (): void => {...}`

Reply:
Ahh, noted and thanks!